cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpExecutionTaskStrategy.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.executor.strategy;
+
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionProposal;
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionTask;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import org.apache.kafka.common.Cluster;
+import org.apache.kafka.common.PartitionInfo;
+
+
+public class PostponeUrpExecutionTaskStrategy implements ExecutionTaskStrategy {
+
+  private boolean isDataMoveOnUrp(Cluster cluster, ExecutionTask task) {
+    PartitionInfo partitionInfo = cluster.partition(task.proposal().topicPartition());
+    return partitionInfo.inSyncReplicas().length != partitionInfo.replicas().length;
+  }
+
+  @Override
+  public Map<Integer, SortedSet<ExecutionTask>> applyStrategy(Collection<ExecutionTask> executionTasks, Cluster cluster) {
+    Comparator<ExecutionTask> taskComparator = (ExecutionTask task1, ExecutionTask task2) -> isDataMoveOnUrp(cluster, task1) ?
```

Comment:
The comparator logic is a little hard to understand -- would you clarify it with a comment?